### PR TITLE
feat(internet-header): initialize package

### PR DESCRIPTION
**Pull request by gfellerph**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/documentation/cypress/e2e/breadcrumbs.cy.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        .should('to.have.attr', 'href', '/a/b/c');
+    });
+  });
+
+  describe('open/close overlay buttons', () => {
+    beforeEach(() => {
+      prepare('internet-header-breadcrumbs', 'Default');
+      cy.get('div.breadcrumbs').as('breadcrumbs');
+      cy.intercept(
+        'https://post.ch/de/kundencenter/onlinedienste/standorte-und-oeffnungszeiten/info?modal=true',
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/breadcrumbs.cy.ts:79]: That's why we intercept the call and provide a mock page.

File: packages/documentation/cypress/e2e/header.cy.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    it('renders', () => {
+      cy.get('swisspost-internet-header').should('have.class', 'hydrated');
+    });
+
+    it(`has title 'CWF Internet Header'`, () => {
+      cy.get('h1').should('contain.text', 'CWF Internet Header');
+    });
+
+    it(`has nav item 'Briefe versenden' selected`, () => {
+      cy.changeArg('active-route', 'https://post.ch/de/briefe-versenden');
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/header.cy.ts:18]: That's just to set the highlights of the active nav entry, no external calls being made.

File: packages/documentation/cypress/e2e/header.cy.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      cy.get('swisspost-internet-header')
+        .shadow()
+        .find('post-main-navigation')
+        .shadow()
+        .find('.main-link.active')
+        .should('contain.text', 'Briefe versenden');
+    });
+
+    it(`has nav item 'Briefe versenden' selected (works with includeShadowDom: true)`, () => {
+      cy.changeArg('active-route', 'https://post.ch/de/briefe-versenden');
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/header.cy.ts:28]: That's just to set the highlights of the active nav entry, no external calls being made.

File: packages/documentation/cypress/fixtures/pages/en/every-language-set.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        /* For header width checks */
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <swisspost-internet-header
+      project="ekp"
+      environment="int01"
+      language="it"
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/fixtures/pages/en/every-language-set.html:23]: Yes, it's the purpose of the test for order of importance.

File: packages/documentation/cypress/e2e/breadcrumbs.cy.ts [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    cy.get('@breadcrumbs')
+      .find('div.overlay')
+      .should('exist')
+      .find('div.overlay-container')
+      .should('have.class', 'loaded');
+
+    cy.get('@breadcrumbs')
+      .find('div.overlay')
+      .trigger('keydown', { eventConstructor: 'KeyboardEvent', force: true, key: key })
+      .wait(500);
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/breadcrumbs.cy.ts]: Removed it, does seem to be unnecessary.

File: packages/documentation/cypress/e2e/footer.cy.ts [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+                  { url: 'https://web.dev', text: 'web.dev', target: '_blank' },
+                  { url: 'https://developer.mozilla.org', text: 'MDN Web Docs', target: '_blank' },
+                  { url: 'https://getbootstrap.com', text: 'Bootstrap', target: '_blank' },
+                  { url: 'https://google.com', text: 'Google', target: '_blank' },
+                  { url: 'https://microsoft.com', text: 'Microsoft', target: '_blank' },
+                ],
+              },
+            },
+          },
+        };
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/footer.cy.ts:38]: I don't think so, it's a config specific to this test.

File: packages/documentation/cypress/e2e/main-navigation.cy.ts [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+                  },
+                  {
+                    title: 'Deeper nav',
+                    linkList: [{ url: url + '/some/levels/deeper', title: 'Deeper nav' }],
+                  },
+                ],
+              },
+            ],
+          },
+        },
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/main-navigation.cy.ts:81]: I don't think so, it's a config specific to this test.

File: packages/documentation/cypress/e2e/main-navigation.cy.ts [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+                flyout: [
+                  {
+                    title: 'Custom Nav',
+                    linkList: [{ url, title: expectedText }],
+                  },
+                ],
+              },
+            ],
+          },
+        },
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/main-navigation.cy.ts:112]: I don't think so, it's a config specific to this test.

File: packages/documentation/cypress/e2e/os-flyout.cy.ts [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+        {
+          title: 'Title',
+          linkList: [
+            {
+              title: 'Additional Link',
+              href: '/link',
+            },
+          ],
+        },
+      ],
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/os-flyout.cy.ts:29]: I don't think so, it's a config specific to this test.

File: packages/documentation/cypress/e2e/os-flyout.cy.ts [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/os-flyout.cy.ts:52]: I don't think so, it's a config specific to this test.

File: packages/documentation/cypress/e2e/os-flyout.cy.ts [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/os-flyout.cy.ts:88]: I don't think so, it's a config specific to this test.

File: packages/documentation/cypress/e2e/os-flyout.cy.ts [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+                },
+                {
+                  title: 'Deeper nav',
+                  linkList: [{ url: '/some/levels/deeper', title: 'Deeper nav' }],
+                },
+              ],
+            },
+          ],
+        },
+      },
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/os-flyout.cy.ts:114]: I don't think so, it's a config specific to this test.

File: packages/documentation/cypress/e2e/os-flyout.cy.ts [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+        {
+          title: 'Title',
+          linkList: [
+            {
+              title: 'Additional Link',
+              href: '/link',
+            },
+          ],
+        },
+      ],
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/cypress/e2e/os-flyout.cy.ts:135]: I don't think so, it's a config specific to this test.

File: packages/documentation/.storybook/main.js [hunk 14]
```diff
@@ -17,14 +17,15 @@ module.exports = {
       },
     },
   ],
-  staticDirs: ['../public'],
+  staticDirs: ['../public', './'],
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/.storybook/main.js:20]: Does not seem to be necessary anymore. I removed it.

File: packages/documentation/.storybook/preview.scss [hunk 15]
```diff
@@ -108,4 +108,5 @@
       line-height: inherit;
     }
   }
+
```
Comment (oliverschuerch) [inline packages/documentation/.storybook/preview.scss:111]:
```suggestion
```

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**